WG-TEMPLATE.md

```diff
@@ -0,0 +1,47 @@
+# OCI Working Group Template
+
+The following is a template that the TOB and/or proposers of a new OCI Working Group may use to document the intended scope and activities of the OCI Working Group following its establishment. {Bracketed sections below} should be filled in with the Working Group's specific details.
+
+# {WG NAME} OCI Working Group - Governance Charter
+
+This document describes the basic governance principles for the {WG NAME} Working Group (the “WG”).
+
+The WG operates as an OCI Working Group under the [Open Container Initiative (OCI) Charter](https://github.com/opencontainers/tob/blob/master/CHARTER.md), which describes the responsibilities of the OCI Technical Oversight Board (the "TOB”). The WG is established by the TOB as an OCI Working Group pursuant to the OCI Charter. Accordingly, the WG will operate in accordance with the OCI Charter and OCI's other policies and procedures, supplemented by the details below.
+
+## Purpose
+
+{Briefly describe the purpose of the WG.}
+
+## Scope
+
+* {Describe the activities and work that are in scope for the WG.}
+
+## Out of Scope
+
+* {Where helpful for clarity or to avoid confusion, describe any activities or topics that are out of scope for the WG.}
+
+## Intended work product
+
+* {Describe the intended work product and outputs of the WG, particularly indicating whether they consist of specification(s) and/or project(s).}
+
+## Governance
+
+* **Working Group**:
+  * The TOB is establishing the WG as an OCI Working Group, pursuant to section 6(p) of the OCI Charter.
+* **Maintainers**:
```

Comment:
The charter section uses "owners" as term to represent this role. The owners of a working group are approved as part of the working group. I'm not sure I understand what "appoint" means in that context. Likewise, for making changes to a working group after it is approved, we should define the process as the owners request a change (in Amendments most likely). Any change requires the same level of approval as the initial proposal. It doesn't make sense to explicitly say that the TOB would remove or appoint new maintainers(/owners), the TOB would be responsible for either disbanding the working group or approving changes proposed by the owners, which could include changes to the owner or sponsor roles.

Reply:
Thanks @dmcgowan -- all good points and thanks for clarifying. I've tried to update in the follow-up commit and clarify the differences between "owners" and "maintainers". Grateful if you can take a look and see if this works for you, or if there are further clarifications that should be added.